frontend/src/components/Admin/DevPortfolio/DevPortfolioDetails.tsx

```diff
@@ -0,0 +1,236 @@
+import { useState, useEffect } from 'react';
+import { Container, Header, Icon, Table } from 'semantic-ui-react';
+import DevPortfolioAPI from '../../../API/DevPortfolioAPI';
+import styles from './DevPortfolioDetails.module.css';
+
+type Props = {
+  uuid: string;
+};
+
+const portfolio = {
+  uuid: 'test test',
+  name: 'Dev Portfolio #1',
+  earliestValidDate: new Date().getTime(),
+  deadline: new Date().getTime(),
+  submissions: [
+    {
+      member: {
+        firstName: 'Jackson',
+        lastName: 'Staniec',
+        netid: 'jks273'
+      },
+      openedPRs: [
+        {
+          url: 'https://github.com/cornell-dti/idol/pull/288',
+          status: 'invalid',
+          reason: 'hello world'
+        }
+      ],
+      reviewedPRs: [
+        {
+          url: 'https://github.com/cornell-dti/idol/pull/288',
+          status: 'invalid',
+          reason: 'hello world'
+        }
+      ]
+    },
+    {
+      member: {
+        firstName: 'Henry',
+        lastName: 'Li',
+        netid: 'hl738'
+      },
+      openedPRs: [
+        {
+          url: 'https://github.com/cornell-dti/idol/pull/288',
+          status: 'valid',
+          reason: 'hello world'
+        },
+        {
+          url: 'https://github.com/cornell-dti/idol/pull/288',
+          status: 'valid',
+          reason: 'hello world'
+        }
+      ],
+      reviewedPRs: [
+        {
+          url: 'https://github.com/cornell-dti/idol/pull/288',
+          status: 'valid',
+          reason: 'hello world'
+        }
+      ]
+    },
+    {
+      member: {
+        firstName: 'Riya',
+        lastName: 'Jaggi',
+        netid: 'rj356'
+      },
+      openedPRs: [
+        {
+          url: 'https://github.com/cornell-dti/idol/pull/288',
+          status: 'valid',
+          reason: 'hello world'
+        },
+        {
+          url: 'https://github.com/cornell-dti/idol/pull/288',
+          status: 'invalid',
+          reason: 'hello world'
+        },
+        {
+          url: 'https://github.com/cornell-dti/idol/pull/288',
+          status: 'invalid',
+          reason: 'hello world'
+        }
+      ],
+      reviewedPRs: [
+        {
+          url: 'https://github.com/cornell-dti/idol/pull/288',
+          status: 'invalid',
+          reason: 'hello world'
+        },
+        {
+          url: 'https://github.com/cornell-dti/idol/pull/288',
+          status: 'invalid',
+          reason: 'hello world'
+        }
+      ]
+    }
+  ]
+};
+
+const DevPortfolioDetails: React.FC<Props> = ({ uuid }) => {
+  // const [portfolio, setPortfolio] = useState<DevPortfolio | null>(null);
+
+  // useEffect(() => {
+  //   DevPortfolioAPI.getDevPortfolio(uuid).then((portfolio) => setPortfolio(portfolio));
+  // }, [uuid]);
```

Comment:
This is temporarily commented out to use the mock submissions for testing purposes. This was a bit easier than actually writing mock data to the database. Will uncomment this before merging of course. This will definitely cause the linter to be unhappy while this is commented out tho.